change/@microsoft-teams-js-504be7c5-dd1e-4992-9779-fe2836fa75c1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Added meeting.registerRaiseHandStateChangedHandler",
```

Comment:
Can you please put `meeting.registerRaiseHandStateChangedHandler` in backtick quotes so it is formatted  correctly for our changelog? Thanks!